### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Signing into Azure Storage_Toggle Explorer Pane - Tree View_ScreenReader: The voiceover incorrectly announces, 'Collapse All' and 'Refresh All' as links in the 'Explorer' tree view panel.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/AzureStorageExplorer#5760]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_AzureToolsStorageExplorer_May2022;#Azure Tools Storage Explorer;#MAC;#FTP;#DesktopApp;#WCAG4.1.2;#Voiceover;#AINotInScope;

## Environment Details:
Application Name: Azure Tools Storage Explorer 
Application Version: 1.24.0
Mac Version: macOS Monterey 12.3.1
Screen Reader: Voiceover

## Repro Steps:
1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch Azure Tools Storage Explorer.
3. Sign in to Azure.
4. Tab to 'Explorer' pane tree view.
5. Tab till 'Collapse All' and 'Refresh All' button.
6. Turn on voiceover and observe.

## Actual:
The voiceover incorrectly announces, 'Collapse All' and 'Refresh All' as links in the 'Explorer' tree view panel.

## Expected:
The voiceover should announce, 'Collapse All' and 'Refresh All' as buttons in the 'Explorer' tree view panel as they do not lead the user into a web page URL destination.

## User Impact:
Screen reader users may find it difficult to perceiver user interface component if incorrect role is assigned.

## Attachment:
![3_Collapse All + Refreh All](https://user-images.githubusercontent.com/98393966/169966075-f143047f-3245-439b-ab68-ea04e4b5560c.gif)

**Comment by JasonYeMSFT** [comment]
This is a duplicate of https://github.com/microsoft/AzureStorageExplorer/issues/5702

**Comment by MRayermannMSFT** [comment]
@JasonYeMSFT  we don't treat this as duplicate, as for accessibility we do 1 bug per OS.

**Comment by JasonYeMSFT** [comment]
We have switched to use real button elements for the "Collapse All" and "Refresh All" buttons.

**Comment by bbarik06** [comment]
Issue is not repro now. Hence, closing the bug. Please refer the attachment.
![5760_Fixed](https://user-images.githubusercontent.com/98393966/176660970-86a9f79c-b77b-41af-a79c-a38d135a8fb8.gif)
